Arduino/MiP_ProMini_Pack_Library/src/mip.h

```diff
@@ -236,7 +236,6 @@ enum MiPClapEnabled
 };
 
 
-
```

Comment:
I would prefer to maintain my original vertical spacing if possible.

Reply:
Yes, sorry, just lost track of vertical spacing as I tried different things.